### DISCUSSION TITLE
Modify Controller Stub for Livewire

### DISCUSSION THREAD

**Discussion by denisonl** [misterdebug/crud-generator-laravel#39] [Ideas]
is It possible to modify the Controller Livewire stub to extend from the Component Class? as livewire doc explains to make Controller Class by executing livewire:make command?

**Comment by misterdebug** [comment]
Can you give me more explanation? 
And why do you suggest this change?

**Reply by denisonl** [reply]
Due to the ControllerLivewire Stub, which extends from Controller class, I came up with an idea to change to Component Class because of executing php artisan livewire:make, generates different code. However, you have a Stub Class "Database.stub" with the same skeleton for livewire implementation.

**Reply by misterdebug** [reply]
I see what you're saying, I'm generating a controller instead of a component because there's a CRUD processing to handle. Search is just a small part of the processing :)
But you have a component for search
